### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by Feliverse**
### Project requirements

- Take a look at the following database diagram for a clinic:

clinic_diagram

- Create a file `named schema_based_on_diagram.sql` where you implement the database from the diagram.
- Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- Remember to add the FK indexes.

**Final result(with a many-to-many table):**
image_720

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -39,6 +39,28 @@ ALTER TABLE animals ADD COLUMN owner_id INT,
...
   FOREIGN KEY (owner_id)
   REFERENCES owners (id);
+  
+  /*Table many-to-many: historial_treatments*/
+
+CREATE TABLE "historial_treatments"(
+    "medical_history_id" INTEGER NOT NULL,
+    "treatment_id" INTEGER NOT NULL
+);
+
```
Comment (Alfredbis29) [inline schema.sql]: - [ ] I noticed that your statement is missing foreign key constraints which are required to ensure data integrity and prevent inconsistencies in the table. I recommend modifying your statement to include the necessary foreign key constraints to link the table to the referenced columns in other tables. and also `medical_history_id` and `treatment_id`, both of which are integers that cannot be NULL. This table does not specify any `foreign key` constraints to reference other tables. :wink: 

_Note:  your table is named `historial_treatments` with two columns: `medical_history_id` and `treatment_id`, and both columns should reference the `id` column in their respective tables. This ensures that only valid medical histories and treatments are included in this table._

I think this how it should look like :point_down: 
```
CREATE TABLE histories_treatments (
    medical_history_id int REFERENCES medical_histories(id),
    treatment_id int REFERENCES treatments(id)
);
```

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @francis3400 , @Feliverse 
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
:heavy_check_mark: Your tables are well structure.
:+1:  Good Job so far
 ## Required Changes 

_Check Under the Review

If you need any suggestion from me I suggest you to use tabindex in your headings paragraph tags to make text . :upside_down_face:
 
 ## Optional suggestions 

- [ ] N/A

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Feliverse) [reply schema.sql]: Ok, We will add that constraint in lines 50 to 53
Thank you

Comment (Feliverse) [reply schema.sql]: Thank you @Alfredbis29 , 
We will add that constraint in lines 50 to 53 in the file: schema_based_on_diagram.sql

Comment (Alfredbis29) [reply schema.sql]: Hello @Feliverse , I hope you are doing well , yes of course

**Review by 1GORDON — COMMENTED** [review]
# Status: INVALID  🚨

Hi @Feliverse & @francis3400  ✋

### This project will be Invalidated and won't affect the number of your weekly reviews

- [ ] Kindly remove the entire live demo section from your readme file since this project is not deployed. This is a general requirement for all projects.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @Feliverse, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**